MEME/MEME/Network/Manager/DetailReviewManager.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  DetailReviewManager.swift
+//  MEME
+//
+//  Created by 임아영 on 3/20/24.
+//
+
+import Foundation
+import Moya
+
+final class DetailReviewManager {
+    typealias API = DetailReviewAPI
+    
+    static let shared = DetailReviewManager()
+    let provider = NetworkProvider<API>(plugins: [NetworkLoggerPlugin()])
+    
+    private init() {}
+    
+    func getDetailReview(reviewId: Int, artistNickName: String, makeupName: String, star: Int, comment: String, reviewImgDtoList: [DetailReviewImage], completion: @escaping (Result<DetailReviewResponse, MoyaError>) -> Void) {
+        provider.request(api: .getDetailReview(reviewId: reviewId, artistNickName: artistNickName, makeupName: makeupName, star: star, comment: comment, reviewImgDtoList: reviewImgDtoList)) { result in
```

Comment:
```suggestion
  func getDetailReview(
    reviewId: Int,
    artistNickName: String,
    makeupName: String,
    star: Int,
    comment: String,
    reviewImgDtoList: [DetailReviewImage],
    completion: @escaping (Result<DetailReviewResponse, MoyaError>) -> Void) {
      provider.request(api: .getDetailReview(
        reviewId: reviewId,
        artistNickName: artistNickName,
        makeupName: makeupName,
        star: star,
        comment: comment,
        reviewImgDtoList: reviewImgDtoList
      )) { result in

```

요롷게 작성하면 어떨까요> 아니면 아예 구조체를 파라미터로 설정해서 하나만 넘겨주게 바꾸는건 어떨까요?